### DISCUSSION TITLE
Deepzoom/Hamburger menu not working

### DISCUSSION THREAD

**Discussion by materiajournal** [thegetty/quire#111] [Get Help]
Hi all, 

In my project, I've got my image files all formatted, in the static/image folder, placed in the figures .yml, and also placed in the article text with shortcode.  That all seems to be working fine in the preview, but for some reason, the zoom function on my images isn't working and neither is the hamburger menu for the site.  It was suggested that this has to do with the javascript? Does anyone have any experience with this problem or know how to fix it?  Thanks so much for any help!

Morgan

**Comment by geealbers** [comment]
Hi! @Erin-Cecele shared some of the Terminal output you had from your call with her. I'm not sure how/if it's related to the possible javascript issues you're having, but I saw it said `gyp: No Xcode or CLT version detected`. There's some info on fixing that error here: https://medium.com/flawless-app-stories/gyp-no-xcode-or-clt-version-detected-macos-catalina-anansewaa-38b536389e8d. Once that's done, try running `quire install` in your project again, and then try `quire preview`.

**Reply by materiajournal** [reply]
Thanks, I'll give that a go!

**Reply by Erin-Cecele** [reply]
@materiajournal Morgan, another thing you can try before re-running `quire install` is to navigate to your `themes` folder and delete the `node_modules` folder. 

<img width="817" alt="Screen Shot 2021-05-03 at 6 46 46 PM" src="https://user-images.githubusercontent.com/62075223/116952616-eaf87280-ac3f-11eb-8fa2-1b654a7f6a8a.png">

**Reply by materiajournal** [reply]
Thanks, @geealbers and @Erin-Cecele. Tried both suggestions and still no dice. It works in my teammates' previews, at least.

**Reply by Erin-Cecele** [reply]
Thanks for letting us know @materiajournal I'll be in touch soon with next steps for us to try.

**Reply by Erin-Cecele** [reply]
Hi @materiajournal. Last we spoke, uninstalling and reinstalling Quire fixed the issues you were having with the sidebar menu and zoom function. Is everything still running smoothly?
